frontend/src/components/layout/navigation/MobileNavigation.tsx

```diff
@@ -63,107 +71,154 @@ export const MobileNavigation = (props: Props) => {
       : styles["not-active"];
 
   return (
-    <nav
-      aria-label={l10n.getString("nav-menu-mobile")}
-      className={`${styles["mobile-menu"]}`}
-    >
-      {/* Below we have conditional rendering of menu items  */}
-      <ul
-        id={`${styles["mobile-menu"]}`}
-        className={`${styles["menu-item-list"]} ${toggleMenuStateClass}`}
+    <>
+      {/* Mask and Phone Header on Mobile */}
+      {isLoggedIn && isFlagActive(runtimeData.data, "phones") ? (
+        <div
+          className={`${styles["nav-mask-phone"]} ${styles["hidden-desktop"]}`}
+        >
+          {/* // Mask Btn */}
+          <Link href={homePath}>
+            <a
```

Comment:
Ah and one more thing, mostly relevant for non-touch input: there's no hover styles for these :)

Reply:
I don't think I added any hover styles 😳